src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -225,32 +231,22 @@ Example:
 
 ```javascript
 // Deploy library LibA, then link LibA to contract B, then deploy B.
-deployer.deploy(LibA);
+async deployer.deploy(LibA);
 deployer.link(LibA, B);
-deployer.deploy(B);
+async deployer.deploy(B);
 
 // Link LibA to many contracts
 deployer.link(LibA, [B, C, D]);
 ```
 
-### deployer.then(function() {...})
 
-Just like a promise, run an arbitrary deployment step. Use this to call specific contract functions during your migration to add, edit and reorganize contract data.
-
-Example:
+Advanced example:
 
 ```javascript
 var a, b;
-deployer.then(function() {
-  // Create a new version of A
-  return A.new();
-}).then(function(instance) {
-  a = instance;
-  // Get the deployed instance of B
-  return B.deployed();
-}).then(function(instance) {
-  b = instance;
-  // Set the new instance of A's address on B via B's setA() function.
-  return b.setA(a.address);
+deployer.then(async function() {
```

Comment:
Same issue as before about `deployer.then` being incorrect. Again, you can just basically get rid of this.